tests/jest/utils/logger/index.test.js

```diff
@@ -0,0 +1,13 @@
+import Logger, {
+  initializeLogger,
+} from '../../../../src/utils/logger';
+
+describe('theme/index', () => {
```

Comment:
```suggestion
describe('logger/index', () => {
```

Reply:
You can tell where i copied this test from. 😅